assets/templates/release.tmpl

```diff
@@ -1,4 +1,16 @@
-<div class="ons-page__container ons-container release">
+<div class="ons-page__container ons-container release" 
+    {{ if .Description.Published }}
+      data-gtm-release-status="published"
+    {{ end }}
+    {{ if .Description.Cancelled }}
+      data-gtm-release-status="cancelled"
+    {{ end }}
+  data-gtm-release-date="{{- .Description.ReleaseDate -}}"
+  data-gtm-release-time="{{- .Description.ReleaseDate -}}"
+  data-gtm-release-date-status="{{- .Description.ReleaseDate -}}"
+  data-gtm-next-release-date="{{- .Description.ProvisionalDate -}}"
```

Comment:
next-release-date should be set to `.Description.NextRelease`

`ProvisionalDate` isn't actually a date at all, but it's used by Florence as a free text field like "Early September" instead of a timestamp.